[2024-10-03T06:26:39.857+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T06:26:39.885+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T06:26:39.897+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [queued]>
[2024-10-03T06:26:39.898+0000] {taskinstance.py:2865} INFO - Starting attempt 4 of 4
[2024-10-03T06:26:39.920+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): warehouse.step_1.extract_transform.categories> on 2024-10-02 00:00:00+00:00
[2024-10-03T06:26:39.926+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T06:26:39.932+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'warehouse.step_1.extract_transform.categories', 'scheduled__2024-10-02T00:00:00+00:00', '--job-id', '13540', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmppw6u_tsk']
[2024-10-03T06:26:39.934+0000] {standard_task_runner.py:105} INFO - Job 13540: Subtask warehouse.step_1.extract_transform.categories
[2024-10-03T06:26:39.935+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6424) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-03T06:26:39.935+0000] {standard_task_runner.py:72} INFO - Started process 6586 to run task
[2024-10-03T06:26:40.006+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.categories scheduled__2024-10-02T00:00:00+00:00 [running]> on host 8c0d519f07a3
[2024-10-03T06:26:40.339+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='warehouse.step_1.extract_transform.categories' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T00:00:00+00:00'
[2024-10-03T06:26:40.340+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T06:26:40.359+0000] {base.py:84} INFO - Retrieving connection 'spark-conn'
[2024-10-03T06:26:40.360+0000] {spark_submit.py:335} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py
[2024-10-03T06:26:40.395+0000] {clientserver.py:505} INFO - Error while sending or receiving.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer
[2024-10-03T06:26:40.397+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T06:26:40.398+0000] {java_gateway.py:1052} INFO - Exception while sending command.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 503, in send_command
    self.socket.sendall(command.encode("utf-8"))
ConnectionResetError: [Errno 104] Connection reset by peer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/java_gateway.py", line 1038, in send_command
    response = connection.send_command(command)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/py4j/clientserver.py", line 506, in send_command
    raise Py4JNetworkError(
py4j.protocol.Py4JNetworkError: Error while sending
[2024-10-03T06:26:40.402+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T06:26:40.414+0000] {spark_submit.py:488} INFO - /home/***/.local/lib/python3.12/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-10-03T06:26:46.157+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SparkContext: Running Spark version 3.5.3
[2024-10-03T06:26:46.159+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-10-03T06:26:46.159+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SparkContext: Java version 17.0.12
[2024-10-03T06:26:46.238+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-03T06:26:46.361+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceUtils: ==============================================================
[2024-10-03T06:26:46.363+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-03T06:26:46.364+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceUtils: ==============================================================
[2024-10-03T06:26:46.365+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SparkContext: Submitted application: ETL Pipeline
[2024-10-03T06:26:46.394+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-03T06:26:46.406+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceProfile: Limiting resource is cpu
[2024-10-03T06:26:46.407+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-03T06:26:46.475+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SecurityManager: Changing view acls to: ***
[2024-10-03T06:26:46.476+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SecurityManager: Changing modify acls to: ***
[2024-10-03T06:26:46.478+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SecurityManager: Changing view acls groups to:
[2024-10-03T06:26:46.479+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SecurityManager: Changing modify acls groups to:
[2024-10-03T06:26:46.481+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-03T06:26:46.864+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO Utils: Successfully started service 'sparkDriver' on port 46129.
[2024-10-03T06:26:46.921+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:46 INFO SparkEnv: Registering MapOutputTracker
[2024-10-03T06:26:47.002+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-03T06:26:47.037+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-03T06:26:47.040+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-03T06:26:47.048+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-03T06:26:47.091+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-24689d9e-59e7-43a5-82ea-a4935a5540fc
[2024-10-03T06:26:47.120+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-03T06:26:47.159+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-03T06:26:47.343+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-03T06:26:47.420+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-10-03T06:26:47.430+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2024-10-03T06:26:47.601+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-03T06:26:47.657+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO TransportClientFactory: Successfully created connection to spark-master/172.20.0.2:7077 after 32 ms (0 ms spent in bootstraps)
[2024-10-03T06:26:47.769+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20241003062647-0002
[2024-10-03T06:26:47.772+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20241003062647-0002/0 on worker-20241003061731-172.20.0.9-38803 (172.20.0.9:38803) with 2 core(s)
[2024-10-03T06:26:47.776+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO StandaloneSchedulerBackend: Granted executor ID app-20241003062647-0002/0 on hostPort 172.20.0.9:38803 with 2 core(s), 1024.0 MiB RAM
[2024-10-03T06:26:47.784+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 46599.
[2024-10-03T06:26:47.784+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO NettyBlockTransferService: Server created on 8c0d519f07a3:46599
[2024-10-03T06:26:47.786+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-03T06:26:47.794+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 8c0d519f07a3, 46599, None)
[2024-10-03T06:26:47.801+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManagerMasterEndpoint: Registering block manager 8c0d519f07a3:46599 with 434.4 MiB RAM, BlockManagerId(driver, 8c0d519f07a3, 46599, None)
[2024-10-03T06:26:47.805+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 8c0d519f07a3, 46599, None)
[2024-10-03T06:26:47.808+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 8c0d519f07a3, 46599, None)
[2024-10-03T06:26:47.847+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:47 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20241003062647-0002/0 is now RUNNING
[2024-10-03T06:26:48.249+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:48 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-10-03T06:26:48.646+0000] {spark_submit.py:488} INFO - Usage: extract_transform.py <function_name> <incremental> <date>
[2024-10-03T06:26:48.653+0000] {spark_submit.py:488} INFO - [2024-10-03T06:26:48.650+0000] {clientserver.py:543} INFO - Closing down clientserver connection
[2024-10-03T06:26:49.520+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO SparkContext: Invoking stop() from shutdown hook
[2024-10-03T06:26:49.523+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-03T06:26:49.549+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO SparkUI: Stopped Spark web UI at http://8c0d519f07a3:4041
[2024-10-03T06:26:49.559+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-10-03T06:26:49.567+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-10-03T06:26:49.615+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-03T06:26:49.650+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO MemoryStore: MemoryStore cleared
[2024-10-03T06:26:49.651+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO BlockManager: BlockManager stopped
[2024-10-03T06:26:49.664+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-03T06:26:49.668+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-03T06:26:49.688+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO SparkContext: Successfully stopped SparkContext
[2024-10-03T06:26:49.689+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO ShutdownHookManager: Shutdown hook called
[2024-10-03T06:26:49.690+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO ShutdownHookManager: Deleting directory /tmp/spark-856787cf-82aa-40e6-bb73-8cc5ff77646b
[2024-10-03T06:26:49.696+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO ShutdownHookManager: Deleting directory /tmp/spark-4758dd03-4799-4063-a23f-9be167082b50
[2024-10-03T06:26:49.702+0000] {spark_submit.py:488} INFO - 24/10/03 06:26:49 INFO ShutdownHookManager: Deleting directory /tmp/spark-4758dd03-4799-4063-a23f-9be167082b50/pyspark-5be885bb-bc72-4b90-949a-cd81e11d6128
[2024-10-03T06:26:49.778+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py. Error code is: 255.
[2024-10-03T06:26:49.787+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=etl_pipeline, task_id=warehouse.step_1.extract_transform.categories, run_id=scheduled__2024-10-02T00:00:00+00:00, execution_date=20241002T000000, start_date=20241003T062639, end_date=20241003T062649
[2024-10-03T06:26:49.788+0000] {taskinstance.py:1563} INFO - Executing callback at index 0: slack_notifier
[2024-10-03T06:26:49.791+0000] {baseoperator.py:405} WARNING - SlackWebhookOperator.execute cannot be called outside TaskInstance!
[2024-10-03T06:26:49.800+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/slack/hooks/slack_webhook.py:42 UserWarning: You cannot override the default channel (chosen by the user who installed your app), username, or icon when you're using Incoming Webhooks to post messages. Instead, these values will always inherit from the associated Slack app configuration. See: https://api.slack.com/messaging/webhooks#advanced_message_formatting. It is possible to change this values only in Legacy Slack Integration Incoming Webhook: https://api.slack.com/legacy/custom-integrations/messaging/webhooks#legacy-customizations
[2024-10-03T06:26:49.811+0000] {base.py:84} INFO - Retrieving connection 'slack_notifier'
[2024-10-03T06:26:50.270+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T06:26:50.271+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 13540 for task warehouse.step_1.extract_transform.categories (Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py. Error code is: 255.; 6586)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark dags/etl_pipeline/tasks/warehouse/components/extract_transform.py. Error code is: 255.
[2024-10-03T06:26:50.309+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-03T06:26:50.335+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T06:26:50.338+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
